docs/_ldi-nifi/processors/archive-file-out.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+parent: LDI NiFi Processors
+title: Archive File Out
+---
+
+# Archive File Out
+
+The Archive File Out is used to write models to files based on a timestamp path property on the model.
+Every file is written to NQuads with the extracted timestamp as name: 2023-11-21-05-05-00-000000000.nq
+When two files have the same name, a sequence nr is added, for example: 2023-11-21-05-05-00-000000000-2.nq
+
+The files are ordered in directories based on the date. For every day, there is one directory.
+For example: 2023-11-21-05-05-00-000000000.nq will be located at  archive-root-dir/2023/11/21.
+
```

Comment:
would explicitly write here that timestampPath property is a must .

format similar to the note on https://informatievlaanderen.github.io/VSDS-Linked-Data-Interactions/ldi-nifi/